outposts/stride/LiquidStakeOutpost.sol

```diff
@@ -51,22 +75,46 @@ contract LiquidStakeOutpost {
     /// with the correct memo to trigger a liquid staking action.
     /// NOTE - on testnet the base denom will be "atevmos"
     /// @param _amount The amount of "aevmos" to be swapped
-    /// @param _receiver The bech32 address of the receiver on the Stride chain
-    function liquidStakeEvmos(uint256 _amount, string memory _receiver) public {
+    /// @param _strideReceiver The bech32 address of the receiver on the Stride chain
+    /// @param _evmosReceiver The bech32 address of the receiver on the Stride chain
+    function liquidStakeEvmos(uint256 _amount, string memory _strideReceiver, string memory _evmosReceiver) public {
         _approveTransfer(_amount, baseDenom);
         Height memory timeoutHeight = Height(100,100);
-        string memory memo = buildLiquidStakeMemo(_receiver);
+        string memory memo = buildLiquidStakeMemo(_strideReceiver, _evmosReceiver);
 
         ICS20_CONTRACT.transfer(
             port,
             channel,
             baseDenom,
             _amount,
             msg.sender,
-            _receiver,
+            _strideReceiver,
+            timeoutHeight,
+            0,
+            memo
+        );
+    }
+
+    /// @dev Transfers the specified amount of "staevmos" to the specified receiver on the Stride chain
```

Comment:
shouldn't it theoretically be `astEvmos`? 😅 not that it really matters, but it would be an atto-stEvmos here